include/beast/detail/write_streambuf.hpp

```diff
@@ -133,11 +133,12 @@ write_streambuf(Streambuf& streambuf, T const& t)
 
 template<class Streambuf, class T0, class T1, class... TN>
 void
-write_streambuf(Streambuf& streambuf, T0&& t0, T1&& t1, TN... tn)
+write_streambuf(Streambuf& streambuf,
+    T0 const& t0, T1 const& t1, TN const&... tn)
 {
-    write_streambuf(streambuf, std::forward<T0>(t0));
-    write_streambuf(streambuf, std::forward<T1>(t1));
-    write_streambuf(streambuf, std::forward<TN>(tn)...);
+    write_streambuf(streambuf, t0);
+    write_streambuf(streambuf, t1);
+    write_streambuf(streambuf, tn...);
 }
```

Comment:
This is very optional.  Actually in the obsessive polishing category...  

You could eliminate this overload: `write_streambuf(Streambuf&)` by changing the above code from:
````
    write_streambuf(streambuf, t0);
    write_streambuf(streambuf, t1);
    write_streambuf(streambuf, tn...);
````
to
````
    write_streambuf(streambuf, t0);
    write_streambuf(streambuf, t1, tn...);
````